Data/Sys/Shaders/default_pre_post_process.glsl

```diff
@@ -4,92 +4,152 @@
 // https://www.unravel.com.au/understanding-color-spaces
 
 // SMPTE 170M - BT.601 (NTSC-M) -> BT.709
-mat3 from_NTSCM = transpose(mat3(
-	0.939497225737661,		0.0502268452914346,		0.0102759289709032,
-	0.0177558637510127,		0.965824605885027,		0.0164195303639603,
-	-0.00162163209967010,	-0.00437400622653655,	1.00599563832621));
+const mat3 from_NTSCM = mat3(
+    0.939497225737661, 0.0177558637510127, -0.00162163209967010,
+    0.0502268452914346, 0.965824605885027, -0.00437400622653655,
+    0.0102759289709032, 0.0164195303639603, 1.00599563832621);
 
 // ARIB TR-B9 (9300K+27MPCD with chromatic adaptation) (NTSC-J) -> BT.709
-mat3 from_NTSCJ = transpose(mat3(
-	0.823613036967492,		-0.0943227111084757,	0.00799341532931119,
-	0.0289258355537324,		1.02310733489462,			0.00243547111576797,
-	-0.00569501554980891,	0.0161828357559315,		1.22328453915712));
+const mat3 from_NTSCJ = mat3(
+    0.823613036967492, 0.0289258355537324, -0.00569501554980891,
+    -0.0943227111084757, 1.02310733489462, 0.0161828357559315,
+    0.00799341532931119, 0.00243547111576797, 1.22328453915712);
 
 // EBU - BT.470BG/BT.601 (PAL) -> BT.709
-mat3 from_PAL = transpose(mat3(
-	1.04408168421813,		-0.0440816842181253,	0.000000000000000,
-	0.000000000000000,	1.00000000000000,			0.000000000000000,
-	0.000000000000000,	0.0118044782106489,		0.988195521789351));
+const mat3 from_PAL = mat3(
+    1.04408168421813, 0.000000000000000, 0.000000000000000,
+    -0.0440816842181253, 1.00000000000000, 0.0118044782106489,
+    0.000000000000000, 0.000000000000000, 0.988195521789351);
 
-float3 LinearTosRGBGamma(float3 color)
+/***** Linear <--> sRGB *****/
+
+float4 LinearTosRGB(float4 c)
+{
+    if (!OptionEnabled(correct_gamma)) {
+        return pow(c, float4(1.0 / game_gamma));
+    }
+
+    else if (OptionEnabled(sdr_display_gamma_sRGB)) {
+        const float a = 0.055;
+
+        c.r = (c.r < 0.0031308) ? (12.92 * c.r) : (pow(c.r, 1.0 / 2.4) * (1.0 + a) - a);
+        c.g = (c.g < 0.0031308) ? (12.92 * c.g) : (pow(c.g, 1.0 / 2.4) * (1.0 + a) - a);
+        c.b = (c.b < 0.0031308) ? (12.92 * c.b) : (pow(c.b, 1.0 / 2.4) * (1.0 + a) - a);
+
+        return c;
+    }
+
+    else {
+        return pow(c, float4(1.0 / sdr_display_custom_gamma));
+    }
+}
+
+float4 sRGBToLinear(float4 c)
```

Comment:
`LinearTosRGB()` `sRGBToLinear()` should be renamed to "GammaToLinear()" or "PerceptionSpaceToLinearSpace()", as sRGB is never used here (the sRGB gamma branch is disabled), and was never used by GameCube/Wii games AFAIK.